src/libs/selectedAccount/selectedAccount.ts

```diff
@@ -313,3 +317,42 @@ export function calculateSelectedAccountPortfolio(
     pending: stripPortfolioState(pendingStateSelectedAccount)
   } as SelectedAccountPortfolio
 }
+
+export const migrateCashbackStatus = (
```

Comment:
Please add some context why this is needed and since which extension version migration is needed.